sentences/sv/_common.yaml

```diff
@@ -59,10 +59,10 @@ expansion_rules:
   är: "(är det | är)"
   brightness: "{brightness} [percent]"
   slå_på: "(tänd | sätt (på | igång | fart på) | slå (på | till) | starta)"
-  slå_av: "(slå (av | från | ifrån) | släck | stäng av | stoppa)"
+  slå_av: "(slå (av | från | ifrån) | släck | stäng (av) | stoppa)"
```

Comment:
Should be "stäng [av]" to make "av" optional.

Reply:
You are correct.